config/config_test.go

```diff
@@ -158,9 +138,6 @@ func makeCfgService(IMUIntegrationEnabled, cloudStoryEnabled bool) resource.Conf
 
 	if cloudStoryEnabled {
 		cfgService.Attributes["cloud_story_enabled"] = true
-		cfgService.Attributes["existing_map"] = "path"
-	} else {
-		cfgService.Attributes["data_dir"] = "path"
```

Comment:
test revert to reflect config validation revert so that we can make config changes without breaking cartographer